docs/installation/cambricon-mlu/online-installation.md

```diff
@@ -0,0 +1,79 @@
+# Online Installation
+
+## Supported Devices
+
+- [x] Cambricon MLUs 
+
+## Supported Platforms
+
+| OS      | Arch           | Supported methods                                                                                                                                 |
+| ------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Linux   | AMD64<br>ARM64 | [pip Installation](#pip-installation) |
+
+## Supported backends
```

Comment:
Do you have verified/supported GPU list like https://docs.gpustack.ai/latest/installation/amd-rocm/online-installation/#supported-devices_1?